src/shared/buttercup/entries.js

```diff
@@ -57,6 +57,12 @@ export function validateEntry(entry) {
     if (fields.filter(field => !field.property).length > 0) {
       errorMessages.push(i18n.t('entry.custom-fields-label-empty-info'));
     }
+
+    const fieldKeys = fields.map(field => field.property);
```

Comment:
This should maybe be (due to attributes):

```javascript
const fieldKeys = fields.reduce((output, field) => {
	if (field.propertyType === "property") {
		output.push(field.property);
	}
	return output;
}, []);
```

Otherwise if an attribute of the same name as a property exists, this will fail.